app/controllers/session_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:email])
+
+    if user&.authenticate(params[:password])
+
+      flash[:notice] = "Welcome #{user.email} back, enjoy your time."
+      remember_token = Digest::SHA256.hexdigest params[:email]
```

Comment:
As indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/before_create_requirement.png) we should user `SecureRandom.urlsafe_base64` to generate a random string instead of using the user email.
We can place the methods for generating the remember token in the User model.